staking/types/validator.go

```diff
@@ -168,7 +187,8 @@ func (v *Validator) SanityCheck() error {
 	}
 
 	// MinSelfDelegation must be >= 1 ONE
-	if !v.Banned && v.MinSelfDelegation.Cmp(big.NewInt(denominations.One)) < 0 {
+	if v.EPOSStatus != effective.Banned &&
```

Comment:
do we have a realistic number of MinDelegation?

Reply:
One one